### ISSUE TITLE
`cargo run --example image -- list` fails

### ISSUE THREAD

**Issue by rukai** [vv9k/docker-api-rs#26]
- Crate version: latest git
- OS: arch linux
- Output of running `docker version` on the command line:
```
Client:
 Version:           20.10.18
 API version:       1.41
 Go version:        go1.19.1
 Git commit:        b40c2f6b5d
 Built:             Sat Sep 10 11:31:10 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.18
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.19.1
  Git commit:       e42327a6d3
  Built:            Sat Sep 10 11:30:17 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.6.8
  GitCommit:        9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6.m
 runc:
  Version:          1.1.4
  GitCommit:
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

When running the image example in the repo I get the following:
```
rukai@memes examples$ cargo run --example image -- list
   Compiling docker-api v0.11.0 (/home/rukai2/Projects/Crates/docker-api-rs)
    Finished dev [unoptimized + debuginfo] target(s) in 0.79s
     Running `/home/rukai2/Projects/Crates/docker-api-rs/target/debug/examples/image list`
Error: invalid value: integer `-1`, expected usize at line 1 column 17
```

Examples like: `cargo run --example container -- list` succeed for me.

**Comment by vv9k** [comment]
I changed the default generated integer type to be `isize` as this was causing this exact  issue on `ImageSummary::containers` field. After fixing that I also noticed that the `ImageSummary::labels` field can be nullable contrary to what the swagger spec says so I added deserialization from Option -> HashMap::default so that it always has value.

Before this changes when running `cargo run --example container -- list` I was getting:
```
Error: invalid value: integer `-1`, expected usize at line 1 column 17
```
And later:
```
Error: invalid type: null, expected a map at line 1 column 131
```

Now it correctly lists images:
```
---------------------------------
Created: 1662075995
Id: sha256:2dc39ba059dcd42ade30aae30147b5692777ba9ff0779a62ad93a74de02e3e1f
Repo tags: ubuntu:latest
Labels:

---------------------------------
Created: 1659403194
Id: sha256:07d9246c53a664dde2b72c3d531d19dff4205a02bdd0e9612a5500aa51b86308
Repo tags: debian:latest
Labels:

---------------------------------
Created: 1657211994
Id: sha256:523ffac7fb2e245e5e7c407b9f7377be9c6c3bf03d380981168311f49030da17
Repo tags: rocky:latest,rockylinux/rockylinux:latest
Labels:
 - io.buildah.version=1.23.1
```

**Comment by Darrilla** [comment]
@vv9k Can you tell us how to get this fix. I'm running into this issue in my own code. See Issue #31

**Comment by vv9k** [comment]
I'll close this as it got fixed in 0.12.0 as mentioned by @Darrilla, if it still persists please reopen.
